src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +56,35 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSessionServiceImpl.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService = (AuthenticationService) injector
+                .getInstance(AuthenticationServiceImpl.class);
+
+        try {
+            authenticationService.register("bob.marley@gmail.com", "password");
```

Comment:
Please handle the RegistrationException and AuthenticationException more gracefully. Throwing a RuntimeException makes it harder to track the actual place where the original exception was thrown. Also, it's better practice to log the exception instead of re-throwing it.